docs/design/w3c/w3c-representation.md

```diff
@@ -0,0 +1,382 @@
+## Design for W3C representation of AnonCreds credentials and presentation
+
+Currently `anoncreds-rs` library only provides support for Indy styled AnonCreds credentials matching to the [specification](https://hyperledger.github.io/anoncreds-spec/). 
+
+This design document proposes extending of `anoncreds-rs` library to add support for AnonCreds W3C representation of verifiable credential and presentation described in the [document]().
+
+### Goals and ideas
+
+* Use Indy styled credentials to generate W3C AnonCreds credentials and presentations
+  * Credentials conversion:
+    * Convert Indy styled AnonCreds credentials into W3C form
+    * Convert W3C styled AnonCreds credentials into Indy form
+  * Presentation conversion (Optional):
```

Comment:
I'd also say that this should be required.

Reply:
It's marked optional, because it may not be needed if we decide to have duplicate methods for w3c (see Q2) which is a recommended approach. So, instead of having `anoncreds_presentation_to_w3c`, the holder can use `anoncreds_w3c_create_presentation` (w3c analogue of `anoncreds_create_presentation`).
Whether the conversion method is needed for the Verifier will depend on how holder and verifier negotiate what version of the protocol (w3c based or legacy Anoncreds based) they will use.